params:
    interpolation_delay: .1
    use_smoothing_interpolator: true
    blending_duration: 2.
    backend_timeout: .1  # 6 backend cycles at dt = 1/60
ros_topics:
    joint_state: /robot/joint_state # Only used by main().
    rmpflow_commands: 
        command: /cortex/arm/command
        ack: /cortex/arm/command/ack
        suppress: /cortex/arm/command/suppress
        interpolated: /cortex/arm/command/interpolated
